.github/actions/deploy/tls_manager.py

```
@@ -0,0 +1,713 @@
+                                      check=False)
+
+            # Delete ConfigMaps
+            self.deployer.run_command([
+                'kubectl', 'delete', 'configmap', 'openshift-service-ca.crt',
+                '-n', self.deployment_namespace
+            ],
+                                      check=False)
+
+            print('✅ TLS certificate cleanup completed')
+
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)